deploy.yml

```diff
@@ -19,6 +21,7 @@ services:
     ports:
       - 5001:5001
     volumes:
-      - ./deploy/devices:/tmp/devices
+      - ./config/device:/tmp/devices
+      - ./scripts:/plugin/scripts
```

Comment:
instead of volume mounting, I'd suggest just adding the scripts directly to the image